lib/src/arithmetic/floating_point/floating_point_adder_round.dart

```diff
@@ -309,9 +309,9 @@ class FloatingPointAdderRound extends FloatingPointAdder {
     final isR = (deltaFlopped.gte(Const(2, width: delta.width)) |
             ~effectiveSubtractionFlopped)
         .named('isR');
+    final infExponent = outputSum.inf(sign: largerSignFlopped).exponent;
```

Comment:
what does this change do? it looks equivalent?

Reply:
It's equivalent.  I didn't pull from main correctly and ended up reimplementing the check for infinity in a slightly different but equivalent way.